### PR TITLE
Check unmounter before Assignment in TestPluginNewUnmounter

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
Check unmounter is not nil before Assignment in TestPluginNewUnmounter
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
/assign @jsafrane
thanks very much!

File: pkg/volume/csi/csi_plugin_test.go [hunk 1]
```diff
@@ -226,18 +226,17 @@ func TestPluginNewUnmounter(t *testing.T) {
...
 
 	if err != nil {
 		t.Fatalf("Failed to make a new Unmounter: %v", err)
 	}
-
-	if csiUnmounter == nil {
+	if unmounter == nil {
 		t.Fatal("failed to create CSI Unmounter")
 	}
+	csiUnmounter := unmounter.(*csiMountMgr)
```
Comment (davidz627) [inline pkg/volume/csi/csi_plugin_test.go]: Correct me if I'm wrong, but I'm not sure that moving this in this way actually changes anything. The logic is still exactly the same except now the error message on line 234 is not accurate.

**Comment by jianglingxia** [comment]
/cc @jsafrane
thanks

**Comment by jsafrane** [comment]
I don't think that this PR is useful, it IMO does to add any new logic and it won't help us to catch errors that aren't already covered by existing test.

**Comment by jianglingxia** [comment]
ok，thanks，I will close it!

**Pull request closed without merging**